### PR TITLE
Modifying samples

### PR DESCRIPTION
**Pull request by shahabl**
Bluemix: Define to avoid using hardware sensors, devices
Hello World: Avoid cout not displayed in console in small samples
Other minor changes

Signed-off-by: Shahab Layeghi <shahab.layeghi@intel.com>

### CODE REVIEW

File: analog-in/cpp/analog-in.cpp [hunk 1]
```diff
@@ -30,7 +30,7 @@
  * Suitable ones in the Grove Starter Kit are the Rotary Angle Sensor, Light
  * Sensor, Sound Sensor, Temperature Sensor. Connected to A0
  * TODO Use a platform with analog input capabilities. Intel Galileo, Edison
- * for instnace
+ * for instanace
```
Comment (anjgola) [inline analog-in/cpp/analog-in.cpp]: spelling error

File: azure-http/sample.json [hunk 2]
```diff
@@ -19,7 +19,7 @@
          "Gateway 32-Bit",
          "Gateway 64-Bit"
       ],
-      "os": ["yocto", "wrs_32", "wrs_64"]
+      "os": ["yocto", "wrs_32"]
```
Comment (anjgola) [inline azure-http/sample.json]: Shahab, just for my knowledge...why was wrs_64 removed?

File: bluemix-flame-detect/README.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
+# Bluemix flame detect Sample
+
+
+## Introduction
+This sample demonstrates using IBM® Bluemix™ to store information gathered be a sensor in a 
+clould service.
```
Comment (anjgola) [inline bluemix-flame-detect/README.md]: typo 'clould'. Sentence is bit confusing and can be interpreted as if the sensor is in the cloud. Maybe rephrase it to end as 'gathered by a sensor.'

File: bluemix-flame-detect/README.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+[paho mqtt C++ Client](https://www.eclipse.org/paho/clients/cpp/)
+
+You can build the paho mqtt library from source, or use [this link](https://www.eclipse.org/downloads/download.php?file=/paho/1.3/eclipse-paho-mqtt-c-unix-1.2.0.tar.gz) to download a pre-built library. After expanding the zipped file, copy the 'libpaho-mqtt3c.so.1' into 
+/usr/lib folder on your target system.
+
+## Setup
+
+Refer to [this page](https://console.bluemix.net/) to get started on IBM®  Bluemix™.
+
+Create a new project on Intel(R) System Studio. After choosing the Bluemix Quickstart sample, it'll ask you for a 'Device ID'. Enter something so you can identify messages from your device in Bluemix console.
```
Comment (anjgola) [inline bluemix-flame-detect/README.md]: Can you explain the 'enter something' part. It is not clear if we are asking user to enter the device id obtained from bluemix or create their own or enter arbitary value. 

Note : this comment might reflect my lack of bluemix setup knowledge. If needed, we can redirect them to : https://github.com/intel-iot-devkit/iot-samples-cloud-setup/blob/master/bluemix-mqtt.md

File: hello-world/c/main.c [hunk 5]
```diff
@@ -21,10 +21,37 @@
  * WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
  */
 
+/*
+ * Copyright (c) 2016 Intel Corporation.
```
Comment (anjgola) [inline hello-world/c/main.c]: Update copyright

File: azure-http/sample.json (hunk 2 above)
Comment (shahabl) [reply azure-http/sample.json]: In wrs_64, Azure IoT library cannot currently be installed since one of the dependencies is not signed.

**Comment by shahabl** [comment]
Made the changes, thank you anjgola

File: azure-http/sample.json (hunk 2 above)
Comment (anjgola) [reply azure-http/sample.json]: Okay. Is that something azure would do? Is there a JIRA to track that?

Comment (shahabl) [reply azure-http/sample.json]: I'm not sure who should fix it, but I'll follow up in a separate task.
